modules/e2ee/E2EEContext.js

```diff
@@ -53,6 +59,19 @@ export default class E2EEcontext {
         });
     }
 
+    /**
+     * Cleans up all state associated with the given participant. This is needed when a
+     * participant leaves the current conference.
+     *
+     * @param {string} participantId - The participant that just left.
+     */
+    cleanup(participantId) {
```

Comment:
didn't you say we don't cleanup earlier today? Or is this just adding the cleanup

Reply:
Oh, looks like past saghul was smart enough to solve it already!